providers/alternatives.rb

```diff
@@ -38,8 +38,9 @@
         description = "Add alternative for #{cmd}"
         converge_by(description) do
           Chef::Log.debug "Adding alternative for #{cmd}"
-          shell_out("rm /var/lib/alternatives/#{cmd}")
-          install_cmd = shell_out("#{alternatives_cmd} --install #{bin_path} #{cmd} #{alt_path} #{priority}")
+          Chef::Log.debug "Runnning #{alternatives_cmd} --install #{bin_path} #{cmd} #{alt_path} #{priority}"
+
+          install_cmd = shell_out("sudo #{alternatives_cmd} --install #{bin_path} #{cmd} #{alt_path} #{priority}")
```

Comment:
yes, ooops I thought that got removed. I am not running this as root. I am testing it out and executing individual commands manually to see what it does.